MANIFEST.md

```diff
@@ -0,0 +1,16 @@
+Manifest 
+---------
+
+Celem naszych spotkań jest poszerzanie wiedzy i wspólne spędzanie czasu w przyjaznej atmosferze. W związku z tym oczekujemy, że wszyscy bez wyjątku będą darzyć się wzajemnym szacunkiem.
+
+Jako społeczność oczekujemy, by każdy z nas stosował się do przedstawionych poniżej zasad, ponieważ uważamy, że pozwoli to wszystkim komfortowo i bezstresowo uczestniczyć w życiu naszej grupy:
+
+* wroc.py jako miejsce przyjazne dla wszystkich pozostaje przestrzenią wolną od formułowania publicznych ocen w kwestiach światopoglądowych, zwłaszcza tych związanych z płcią, preferencjami seksualnymi, budową ciała, rasą lub wyznaniem.
```

Comment:
@pwilczynskiclearcode Nie widzę nic złego w tym, aby ludziom czasami przypomnieć o tym, jak się poprawnie zachowywać. Kupując bilet na koncert czy wchodząc do centrum handlowego zgadzasz się na regulamin organizatora/właściciela centrum, z którego to 90% jest po prostu reiteracją istniejącego już prawa. Taki zwyczaj, handlujmy z tym. 8-)